src/pal/src/numa/numa.cpp

```diff
@@ -741,7 +741,7 @@ GetCurrentProcessorNumberEx(
     ENTRY("GetCurrentProcessorNumberEx(ProcNumber=%p\n", ProcNumber);
 
     DWORD cpu = GetCurrentProcessorNumber();
-    _ASSERTE(cpu < g_possibleCpuCount);
+    _ASSERTE(cpu < (size_t)g_possibleCpuCount);
```

Comment:
These numbers are not sizes of memory blocks. They are regular integers. Casting the other side to `int` would be more appropriate.

Reply:
(multiple places)